src/components/QuranReader/ReaderContainer.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from 'react';
+import { useSelector } from 'react-redux';
+import { NOTES_SIDE_BAR_DESKTOP_WIDTH } from 'src/styles/constants';
+import { selectNotes } from 'src/redux/slices/QuranReader/notes';
+import styled from 'styled-components';
+import Notes from './Notes/Notes';
+import ContextMenu from './ContextMenu';
+
+interface Props {
+  children: ReactNode | ReactNode[];
+}
+
+const ReaderContainer: React.FC<Props> = ({ children }) => {
+  const isSideBarVisible = useSelector(selectNotes).isVisible;
+  return (
+    <>
+      <ContextMenu />
+      <Container isSideBarVisible={isSideBarVisible}>{children}</Container>
```

Comment:
re: from our discussion earlier. isSideBarVisible should not be passed from the parent container.

Ideally we keep the props minimal and simple. 

And thank you for the container refactor!